py-utils/src/utils/msg_bus/message_bus/adaptee/confluent_kafka_adaptor.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+from collections import namedtuple
+
+from confluent_kafka.cimpl import Producer, Consumer
+from confluent_kafka.admin import AdminClient, NewTopic, ClusterMetadata
+from message_bus.adaptee import Adaptee
+from message_bus.utils import log_decorator
+
+ConsumerRecord = namedtuple("ConsumerRecord",
+                            ["topic", "message", "partition", "offset", "key"])
+
+
+class ConfluentAdaptee(Adaptee):
+
+    def __init__(self, config):
+        try:
+            self.config = config
+            self.mapper = {}
+        except Exception as e:
+            print(e)
+
+    def create_admin(self):
+        config = self.config['client'][0]
+        # self.admin = AdminClient(**config)

```

Comment:
Remove all the commented code once done. Until this is done, patch can not be merged.

Reply:
Removed